benchmark/metrics/test/test_cputime.py

```diff
@@ -204,7 +213,8 @@ def filter(node_cpu_times):
             for process_times in node_cpu_times.values():
                 if process_times:
                     for process in process_times:
-                        process_times[process] = 0
+                        if process != WALLCLOCK_LABEL:
+                            process_times[process] = 0
```

Comment:
Since it's test code, I'd rather keep it verbose and obvious and independent of external libraries.